tests/components/item_grouping_util.spec.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { expect } from "chai";
```

Comment:
do we really need a new dependency for it? can't we use expect that is already used allover the current tests?